pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -256,6 +278,9 @@ func (a *Bootstrap) addSystemdUnits(uri string, templateData *bootstrapTemplateD
 		"chown-gatewayd-key.service":      {},
 		"systemd-journal-gatewayd.socket": {},
 		"approve-csr.service":             {},
+		// baremetal platform services
+		"keepalived.service": {},
```

Comment:
Wouldn't this break for platforms other than `baremetal`?
You should probably append to the `enabled` map in an if branch.

Reply:
That's a good question. When I first saw this, I thought the code below this only enabled units if they exist but on a second look that doesn't appear to be the case. The other platforms tests are passing, but there might be errors in the journal on bootstrap if it's trying to enable these non-existent service. @hardys should we make these two services specific to baremetal?

Reply:
Alternatively you could use terraform to define and enable the services.

Reply:
No I think this is Ok, I previously discussed with @staebler and this was the suggested approach, the code only checks whether to enable units that are in the configuration e.g we call addSystemdUnits twice in the baremetal case (with two different directories), and in all other cases those extra keys are just ignored because we never look for those names in the map.

Reply:
Also see below the code does:

```
    if _, ok := enabled[name]; ok
    ...
```

So we don't care if something not-name is in the enabled map, only if you try to lookup a service that doesn't exist.